raiden_libs/network/matrix/client.py

```diff
@@ -377,8 +377,15 @@ def _handle_response(self, response):
             for event in sync_room['account_data']['events']:
                 room.account_data[event['type']] = event['content']
 
-        for event in response['account_data']['events']:
-            self.account_data[event['type']] = event['content']
+        # Handle presence after rooms
+        for presence_update in response['presence']['events']:
+            for callback in self.presence_listeners.values():
+                self.call(callback, presence_update)
+
+        if self._first_sync:
+            for event in response['account_data']['events']:
+                self.account_data[event['type']] = event['content']
```

Comment:
Could you please add a brief description on what this is and why it is only being done once?